octopose/deploy.py

```diff
@@ -32,11 +32,14 @@
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument('infile', nargs='?', type=argparse.FileType('rb'), default=sys.stdin)
+    parser.add_argument('infile', nargs='?', type=argparse.FileType('rb'), default=sys.stdin,
+                        help='Supply a .json manifest file as an input')
     parser.add_argument('-e', '--environment', default="local", type=str,
                         help="Supply the environment that you would like to deploy to.")
-    parser.add_argument('--wait', action="store_true", help="Wait and poll for deploy")
-    parser.add_argument('--force', action='store_true', help='Force deployment')
+    parser.add_argument('--wait', action="store_true",
+                        help="Wait and poll for deploy. Octopose will continually poll the Octopus Deploy Tasks till they are complete")
+    parser.add_argument('--force', action='store_true',
+                        help='Force deployment. Ensures the package is re-deployed even if the selected version is already deployed into the target environment')
     parser.add_argument('-v', '--verbose', action='store_true', help="Do not suppress logging from deploy scripts.")
```

Comment:
I would update this message ... verbose says we are getting extra debug info IMO ... up to you guys. But would change to something along the lines of `Verbose logging gives extra contextual output.`
Also, what is the default on this arg?  I get confused with this one ... by default we should not log at verbose level.

Reply:
Default is do not log at the verbose level